src/instantsend.cpp

```diff
@@ -99,7 +101,7 @@ bool CInstantSend::ProcessTxLockRequest(const CTxLockRequest& txLockRequest, CCo
 {
     LOCK(cs_main);
 #ifdef ENABLE_WALLET
-    LOCK(pwalletMain ? &pwalletMain->cs_wallet : NULL);
+    LOCK(vpwallets[0] ? &vpwallets[0]->cs_wallet : NULL);
```

Comment:
Should be smth like 
```suggestion
    LOCK(vpwallets.empty() ? nullptr : &vpwallets[0]->cs_wallet);
```
here and in similar cases below. But we are going to drop legacy IS in #3020 anyway, so there is no need fixing this (and some `if`s) imo - just wait and rebase :)

Reply:
Yepp, lets merge #3020 first and get rid of changes in instantsend.cpp

Reply:
resolved by rebase